Northstar.CustomServers/mod/scripts/vscripts/mp/_gamestate_mp.nut

```diff
@@ -666,7 +666,8 @@ void function CleanUpEntitiesForRoundEnd()
 	foreach ( entity player in GetPlayerArray() )
 	{
 		ClearTitanAvailable( player )
-		
+		PROTO_CleanupTrackedProjectiles( player )
+		player.SetPlayerNetInt( "batteryCount", 0 ) 
```

Comment:
```suggestion
		player.SetPlayerNetInt( "batteryCount", 0 ) 
```

The `PROTO_CleanupTrackedProjectiles`  should be it's own PR. The rest works and fixes our current problem with batteries ^^

Reply:
Ok, will revert

Reply:
Done now